.vscode/snipsnap.code-snippets

```diff
@@ -0,0 +1 @@
roup().","scope":"javascript,javascriptreact,typescript,typescriptreact"},"consolelog-a7a4472":{"prefix":["base console log"],"body":["console.log(${1:object});"],"description":"Displays a message in the console","scope":"javascript,javascriptreact,typescript,typescriptreact"},"consolelogobject-2197f96":{"prefix":["base console log object"],"body":["console.log('${1:object} :>> ', ${1:object});"],"description":"Displays an object in the console with its name","scope":"javascript,javascriptreact,typescript,typescriptreact"},"consoletrace-33e4543":{"prefix":["base console trace"],"body":["console.trace(${1:object});"],"description":"Prints a stack trace from the point where the method was called","scope":"javascript,javascriptreact,typescript,typescriptreact"},"consolewarn-1ddec9c":{"prefix":["base console warn"],"body":["console.warn(${1:object});"],"description":"Displays a message in the console but also displays a yellow warning icon along with the logged message","scope":"javascript,javascriptreact,typescript,typescriptreact"},"consoleinfo-9aaf424":{"prefix":["base console info"],"body":["console.info(${1:object});"],"description":"Displays a message in the console but also displays a blue information icon along with the logged message","scope":"javascript,javascriptreact,typescript,typescriptreact"},"consoletable-35c244f":{"prefix":["base console table"],"body":["console.table(${1:object});"],"description":"Displays tabular data as a table.","scope":"javascript,javascriptreact,typescript,typescriptreact"},"consoletime-c352d2d":{"prefix":["base console time"],"body":["console.time(${1:object});"],"description":"Sets starting point for execution time measurement","scope":"javascript,javascriptreact,typescript,typescriptreact"},"consoletimeend-edc0765":{"prefix":["base console time end"],"body":["console.timeEnd(${1:object});"],"description":"Sets end point for execution time measurement","scope":"javascript,javascriptreact,typescript,typescriptreact"},"settimeout-1eb369f":{"prefix":["base set timeout"],"body":["setTimeout(() => {\n\t${0}\n}, ${1:delay});"],"description":"setTimeout","scope":"javascript,javascriptreact,typescript,typescriptreact"},"setinterval-25bbd90":{"prefix":["base set interval"],"body":["setInterval(() => {\n\t${0}\n}, ${1:delay});"],"description":"setInterval","scope":"javascript,javascriptreact,typescript,typescriptreact"},"setimmediate-44177fa":{"prefix":["base set immediate"],"body":["setImmediate(() => {\n\t${0}\n});"],"description":"setImmediate","scope":"javascript,javascriptreact,typescript,typescriptreact"},"process-nexttick-9043b4d":{"prefix":["base process next tick"],"body":["process.nextTick(() => {\n\t${0}\n});"],"description":"process nextTick","scope":"javascript,javascriptreact,typescript,typescriptreact"},"insert-use-strict-statement-dbe202c":{"prefix":["base use strict"],"body":["'use strict';"],"description":"insert 'use strict' statement","scope":"javascript,javascriptreact,typescript,typescriptreact"}}
```

Comment:
are you supposed to commit this? isn't this part of your personal workspace?

Reply:
Fixed